tests/phpunit/tests/option/option.php

```diff
@@ -679,7 +680,7 @@ public function test_update_option_should_add_option_when_the_new_value_is_store
 	 */
 	public function data_stored_as_empty_string() {
 		return array(
-			'false'        => array( false ),
+			'false'        => array( '' ),
 			'empty string' => array( '' ),
```

Comment:
```suggestion
			'empty string' => array( '' ),
```
`'false' => array( '' )` is a duplicate of `'empty string' => array( '' )`, just with a different dataset name. We can remove this as `''` is already tested in the `empty string` dataset.